updates/23.sql

```diff
@@ -16,3 +16,5 @@ INSERT INTO datenbank_mapping (Formular, Datenfeld, de_Beschriftung, Feldtyp, Ar
 VALUES ("einzelbeleg", "KritikSelektion", "Textkritische Anmerkung", "select", 0, "einzelbeleg", "KritikID", "selektion_kritik", "einzelbeleg", "Text-critical note", "Note critique sur le texte", "Nota cum textu critico");
 
 ALTER TABLE einzelbeleg ADD KritikID INT DEFAULT -1;
+
+ALTER TABLE einzelbeleg ADD CONSTRAINT FK_EinzelbelegKritik_1 FOREIGN KEY (KritikID) REFERENCES selektion_kritik(ID);
```

Comment:
If you execute `SHOW CREATE TABLE einzelbeleg` you can see that all other existing foreign keys follow a naming scheme, e.g. `einzelbeleg_BearbeitungsstatusID`, `einzelbeleg_EditionID` and so on. Please change the name to `einzelbeleg_KritikID` according to the existing convention.